pkgs/by-name/sl/sleek-grub-theme/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
```

Comment:
```suggestion
, fetchFromGitHub
, banner ? "Grub Bootloader" # can override to your own banner
, style ? "white" # can override to "white" / "dark" / "orange" / "bigSur"
}:
```

Reply:
I think we shoud use `overrideAttrs` instead of `override` here?

Reply:
Now using `stdenv.mkDerivation (finalAttrs: { ... })`  instead of `rec` to solve the `overrideAttrs` issue

Reply:
No `override` is better.

Reply:
Is there any guide document about why "override is better" ?    it's not obvious for me .

Reply:
It's shorter, doesn't require a function. Also look at what other customizable packages use